cmake/utils.cmake

```diff
@@ -138,3 +138,32 @@ function(set_hardening_flags)
         endif()
     endif()
 endfunction()
+
+#
+# Sets the `MACRO_PREFIX_MAP_FLAG` and `DEBUG_PREFIX_MAP_FLAGS` used for `-fmacro-prefix-map` and `-fdebug-prefix-map`
+# with `-Wa,--debug-prefix-map` compiler flags respectively.
+#
+# Uses the `DEV_BUILD` option.
+function(set_prefix_map_flags)
```

Comment:
This doesn't look like an utility function. Why not set the flag in `compiler.cmake`?

Reply:
Fixed: moved the code to a new `cmake/prefix.cmake` module.